distributed/dashboard/components/scheduler.py

```diff
@@ -1721,6 +1730,450 @@ def __del__(self):
         self.scheduler.remove_plugin(self.layout)
 
 
+##########Task Group Graph #########
+class TGroupGraph(DashboardComponent):
+    """
+    Task Group Graph
```

Comment:
Can I ask for a brief summary here.  Maybe just a few lines saying that this uses TaskGroups, lays them out by dependency information, and then incodes progress/duration/memory/type into glyphs.

Reply:
I added some more text, it probably needs to be polished.